backends/tfhe-cuda-backend/cuda/include/pbs/pbs_enums.h

```diff
@@ -3,6 +3,7 @@
 #include <stdint.h>
 enum PBS_TYPE { MULTI_BIT = 0, CLASSICAL = 1 };
 enum PBS_VARIANT { DEFAULT = 0, CG = 1, TBC = 2 };
+enum PBS_MS_REDUCTION_T { NO_REDUCTION = 0, DRIFT = 1, CENTERED = 2 };
```

Comment:
I think you could generate a Rust binding for this enum by placing it in the integer.h file instead. Or we could maybe generate bindings for pbs_enums, and in that case it'd have to be in the extern C section